core/storage/statistics/gae_models.py

```diff
@@ -52,6 +52,16 @@ class StateCounterModel(base_models.BaseModel):
 
     @classmethod
     def get_or_create(cls, exploration_id, state_name):
+        """Gets or creates an entity by exploration_id and state_name.
+
+        Args:
+            exploration_id: str. id of the exploration currently being played.
+            state_name: str
+
+        Returns:
+            entity instance corresponding to the given exploration_id and
```

Comment:
Missing type info. See other "Returns" docstrings for the correct format.

Reply:
done..